SSServer/src/main/resources/installation/start.sh

```diff
@@ -3,4 +3,11 @@ export DB_USER=$1
 export DB_PW=$2
 export DB_URI=$3
 export DB_NAME=$4
-java -jar lib/SSServer-0.0.1-SNAPSHOT.jar --spring.config.location=config/ --spring.profiles.active=$5
\ No newline at end of file
+java -jar lib/SSServer-0.0.1-SNAPSHOT.jar --spring.config.location=config/ --spring.resources.static-locations=web/static --spring.profiles.active=$5
```

Comment:
Here two possibilities one with the --spring.resources.static-locations=web/static path